README.rst

```diff
@@ -37,8 +37,8 @@ To run it, start python shell
 
 .. code:: console
 
-    >>> from SECEdgar.filings import Filing
-    >>> my_filings = Filing(cik='0000320193', filing_type='10-q', count=15) # 10-Q filings for Apple (NYSE: AAPL)
+    >>> from SECEdgar.filings import Filing, FilingType
+    >>> my_filings = Filing(cik='0000320193', filing_type=FilingType.FILING_10K) # 10-Q filings for Apple (NYSE: AAPL)
```

Comment:
Can we add one more interface to `Filing` which will fetch the cik? Our current format for cik constraining here like leading zeros and a lot of users can make mistakes and input wrong cik codes.

What I am thinking of is eliminating the use of string codes and instead use the company name. The new API should accept `Filing(cik=CIK(for:)` where CIK is an instance of a CIK class whose responsibility is to fetch the codes.

or we can keep the above API for backward compatibility and add this new one.

```
apple_cik = CIK(for: "apple").code // return the first code
multi_cik = CIK(for: "apple, facebook, microsoft").codes // This will return a list.
```

Ignore as it is not related to this MR.